wagtail/admin/templates/wagtailadmin/pages/_unsaved_changes_warning.html

```diff
@@ -0,0 +1,12 @@
+{% load wagtailadmin_tags i18n %}
+<li class="footer__container footer__container--hidden footer__save-warning" data-unsaved-warning>
+    <div hidden data-unsaved-type="any" class="footer__save-warning">
+        {% icon name="warning" class_name="default footer__emphasis" %}
+        <div>
+            <b hidden data-unsaved-type="edits">{% blocktrans %}You have <span class="footer__emphasis">unsaved edits</span>{% endblocktrans %}</b>
```

Comment:
Putting classnames in a translatable string could cause some confusion (translators might translate the class name too)

Reply:
Yeah, I can see that, but I'm not quite sure how to get around that, because splitting up eg "You have", "unsaved comments" might not work depending on the language, where the emphasis might need to be on different bits. Do you have any suggestions?

Reply:
You could add the styling to a surrounding tag, like:

```css
.footer__emphasise-span-tags span {
   ...
}
```

```
<b class="footer__emphasise-span-tags">{% blocktrans %}You have <span>unsaved ...
```

Reply:
Good point, thanks! Completely slipped my mind I could do that...